features

```diff
@@ -1 +1,2 @@
 cache-native-yarn-cache=100
+use-heroku-yarn-prune-plugin=20
```

Comment:
@joshwlewis what do you think of enabling this for 20% of users?  yarn 2 & 3 was used in roughly 5% of the builds from the last week.

Reply:
Works for me!